ToolBox/YahooDownloader/YahooChartJsonResponse.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+namespace QuantConnect.ToolBox.YahooDownloader
+{
+
+    public partial class YahooChartJsonResponse
```

Comment:
I think this file is not required? It's not included in the project either

Reply:
Removed